index.bs

```diff
@@ -268,6 +268,54 @@ https://school.register.gov.uk/proof/entry/123/merkle:sha-256
 </pre>
 </div>
 
+## Consistency proof resource ## {#consistency-proof-resource}
+<dl class="resource"><dt>Path</dt><dd>/proof/consistency/{total-entries-1}/{total-entries-2}/{proof-identifier}</dd></dl>
+
+A consistency proof proves the append-only property of a register.
+
+Assuming n is the greatest of total-entries-1 and total-entries-2 and m is the smallest of total-entries-1 and total-entries-2 (m <= n), a consistency proof for a register of n entries and a previous version of the same register with m entries is the information required to prove that the first m entries are equal for both, given a [[#register-proof-resource]] for both versions of the register.
+
+The important characteristic of a consistency proof between two versions of a register is that the client does not need to download the entirety of either to verify consistency between the two.
+
+There may be different kinds of consistency proof available.
+
+<div class="example">
+The following example shows a Merkle-tree-based consistency proof in the [[#json-representation]]:
+
+<pre>
+https://school.register.gov.uk/proof/consistency/1234/1240/merkle:sha-256
+</pre>
+<pre highlight="json">
+{
+  "proof-identifier": "merkle:sha-256",
+  "merkle-consistency-path": [ "MAzvw8AsFqZ8Scuc5IPfj0dzl44jJauaNXuZLQxR3bM=", "TX/kGqrSEgHGvxLwSMyX5al14G48HyPmKbUYK0+wSCE=" ]
+}
+</pre>
+</div>
+
+## Record proof resource ## {#record-proof-resource}
+<dl class="resource"><dt>Path</dt><dd>/proof/record/{total-entries}/{<a href="#field-value">field-value</a>}/{proof-identifier}</dd></dl>
```

Comment:
We don't have a standard way of doing this, but I think it'd be worth marking this as experimental somehow. Maybe open an issue and refer to it here?